content/news/2023-10-21-bevy-0.12/index.md

```diff
@@ -15,6 +15,45 @@ Since our last release a few months ago we've added a _ton_ of new features, bug
 
 <!-- more -->
 
+## PCF Shadow Filtering
+
+<div class="release-feature-authors">authors: @superdump (Rob Swain), @JMS55</div>
+
+Shadow aliasing is a very common problem in 3D apps:
+
+![no pcf](no_pcf.png)
+
+Those "jagged lines" in the shadow are the result of the shadow map being "too small" to accurately represent the shadow from this perspective. The shadow map above is stored in a 512x512 texture.
+
+One "solution" is to bump up the resolution. Here is what it looks like with a 4096x4096 shadow map.
+
+![no pcf high resolution](no_pcf_high.png)
+
+Looking better! However this still isn't a perfect solution. Large shadowmaps aren't feasible on all hardware. And even if they are, you can still encounter this issue if you place an object in the wrong place, or point your light in the wrong direction. You can use Bevy's [Cascaded Shadow Maps](/news/bevy-0-10/#cascaded-shadow-maps) (which are enabled by default) to cover a larger area, with higher detail close to the camera and less detail farther away. However even under these conditions, you will still probably encounter these aliasing issues.
```

Comment:
Large shadow maps take a lot of memory and more importantly rendering time. I think saying hardware capabilities is misleading, it's more about performance.